tests-manual/debugger/README.md

```diff
@@ -0,0 +1,45 @@
+Debugger Command Line Test Programs
+===================================
+
+This directory contains a set of utilities used to manually test the pipeline debugger features. These tests should be run against a live system with the debugger facilities built into it.
```

Comment:
"_with the debugger facilities built into it_:" Is it clear from other documentation how to do this?